packages/k-ramel/doc/LISTENERS.md

```diff
@@ -16,3 +16,63 @@
     <a href="./STORE.md">store</a>
   &nbsp;]
 </p>
+
+## What is a listener?
+You can listen to actions from the redux eventbus with what we call `listeners`.\
+A listener is a predicate wrapped into the `when` function (this is a HoF _Higher order Function_).
+With a listener you call what we call a `reaction`.
+
+You listen (using a `listener` (`when` function)) a specific action from the redux eventbus, then you `react` to it via an other function that we call a reaction.
+
+You can look at [reactions documentation here](./REACTIONS.md).
+
```

Comment:
Maybe explain that you can hot load the listeners and give a link to the react `withListeners` function